docs/BreakingChanges/069 EF no longer throws for QueryViews with specific c.md

```diff
@@ -0,0 +1,30 @@
+## 69: EF no longer throws for QueryViews with specific characteristics
```

Comment:
{
  "Id": "69",
  "Title": "EF no longer throws for QueryViews with specific characteristics",
  "Details": "TODO",
  "Suggestion": "TODO",
  "Link": "https://msdn.microsoft.com/en-us/library/dn720774(v=vs.110).aspx",
  "Markdown": null,
  "ApplicableApis": [
    "Not detectable via API analysis"
  ],
  "Related": null,
  "VersionBroken": "4.5.2",
  "VersionFixed": null,
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": null,
  "Notes": "Probably detectable, but may need some insight into whether included entities are such that they would trigger this.\nSource analyzer status: Not planned",
  "IsRetargeting": false,
  "ImpactScope": 3
}

**Parsed APIs:**
- [ ] Not detectable via API analysis